src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -486,17 +499,14 @@ export class TableInteractorColumnComponent implements OnInit {
     // The line ends at this interactor or on any of its subcomponents
     if (complex.endInteractorIndex != null && complex.endInteractorIndex === interactorIndex) {
 
-      // The line starts before this interactor and ends at this interactor or on any of its subcomponents
-      if (complex.startInteractorIndex < interactorIndex && complex.endInteractorIndex === interactorIndex) {
-        // If the interactor is an expanded subcomplex, and there is any line between the subcomponents, then
-        // the line does not end in this interactor and it must cross through to the subcomponents
-        if (this._enrichedInteractors[interactorIndex].isSubComplex && this._enrichedInteractors[interactorIndex].expanded) {
-          if (complex.startSubComponentIndex != null && complex.endSubComponentIndex != null) {
-            return false;
-          }
+      // If the interactor is an expanded subcomplex, and there is any line between the subcomponents, then
```

Comment:
There was a small bug in `doesLineEndOnInteractorCell` calculating if a line ended in an interactor, on the edge case of complexes with a single cell.

This fixes that bug and simplifies this function that had extra ifs not needed.